vpc/service/assign_addresses_v3.go

```diff
@@ -158,11 +158,11 @@ LIMIT 1
 	err = row.Scan(&ret.az, &ret.vpcID, &ret.accountID, &ret.subnetID, &ret.cidr, &ret.region)
 	if err == sql.ErrNoRows {
 		if len(subnetIDs) == 0 {
-			err = fmt.Errorf("No subnet found matching IDs %s in az %s", subnetIDs, az)
+			err = newNotFoundError(fmt.Errorf("No subnet found matching IDs %s in az %s", subnetIDs, az))
 		} else {
-			err = fmt.Errorf("No subnet found in account %s in az %s", accountID, az)
+			err = newNotFoundError(fmt.Errorf("No subnet found in account %s in az %s", accountID, az))
```

Comment:
If we also log the subnetIDs as part of the error message, it could be useful for aggregation queries, etc?

Reply:
Yea,  this is just a classification thing.